pkg/lifecycle/parser_test.go

```diff
@@ -159,15 +159,15 @@ func Test_sortAssetsVersions(t *testing.T) {
 	dep.sortAssetsVersions()
 
 	// Assert
-	if dep.assetsVersionsMap["key1"][0].Version != "0.0.1" ||
-		dep.assetsVersionsMap["key1"][1].Version != "0.1.0" ||
-		dep.assetsVersionsMap["key1"][2].Version != "1.0.0" {
+	if dep.AssetsVersionsMap["key1"][0].Version != "0.0.1" ||
```

Comment:
We don't need to change this now nor do I know if you even wrote this but for reference, Hungarian notation in Go is a code smell and should be avoided.

Reply:
Fixed and also implemented assert package for unit-testing, it is easier to read. @briandowns